example/js/lib/api/diplomat-runtime.js

```diff
@@ -111,7 +111,7 @@ export class DiplomatBuf {
       if (successfully_unregistered) {
         // leak
       } else {
-        console.error(`Failed to unregister DiplomatBuf at ${ptr}, this is a bug. Either it was never registered (leak), it was already unregistered (failed attempt to double free), or the unregister token was unrecognized (fallback to GC).`);
+        console.error(`Failed to unregister DiplomatBuf at ${this.ptr}, this is a bug. Either it was never registered (leak), it was already unregistered (failed attempt to double free), or the unregister token was unrecognized (fallback to GC).`);
```

Comment:
Question: why is `leak` a member function but `free` is declared inline?

Reply:
no idea!

Reply:
i think they were written in different PRs and I didn't notice the inconsistency

Reply:
Merging this fix but I can look into making this consistent later

Reply:
Oh it's because free() needs additional ctor-time context, leak doesn't.

Reply:
It only seems to be `align` which seems like something that should probably be stored on the instance anyway